tests/conflux/messages.py

```diff
@@ -82,9 +82,13 @@ def deserialize(self, serial):
             return False
         elif serial == b'\x01':
             return True
+        elif serial == b'':
```

Comment:
You can see the RLP change here : `https://github.com/paritytech/parity-common/pull/572`